tools/doc/README.md

```diff
@@ -1,108 +0,0 @@
-Here's how the node docs work.
```

Comment:
might be helpful to keep this?

Reply:
-1 to keep here but +1 to have something like that in `nodejs/api-docs-tool`